srcpkgs/base-files/files/_vkpurge

```diff
@@ -0,0 +1,29 @@
+#compdef vkpurge
+
+_vkpurge() {
+	local line state
+
+	_arguments -C \
```

Comment:
> The option `-C` tells `_arguments` to modify the `curcontext` parameter for an action of the form ‘->state'.  This is the standard parameter used to keep  track  of  the current  context.   Here  it  (and  not the context array) should be made local to the calling function to avoid passing back the modified value and should be initialised to the current value at the start of the function:
> 
>     local curcontext="$curcontext"
> 
> This is useful where it is not possible for multiple states to be valid together.